_people/george_waring.md

```diff
@@ -5,7 +5,7 @@ graduated: 2018
 award: Merit
 gender: male
 course:
-  - International Relations BA Hons
+  - BA Hons International Relations
```

Comment:
Is a BA and a BA Hons the same?

Reply:
I think so, but it we're stripping the degree level for the search function anyway then it's not really an issue.

think you can get a BA Hons in a straight subject but only a BA if you do joint Honours

Reply:
Okay - can we therefore have all the Hons in brackets? (Stripping them out with brackets feels more technically foolproof than without)